rtsan.h

```diff
@@ -0,0 +1,129 @@
+//===-- rtsan.h -------------------------------------------------*- C++ -*-===//
```

Comment:
First off, this is in the very top level of the repo, first level. Do we want to add it to a directory? `include` or `rtsan` or similar?